src/backend/aspen/api/views/pathogens.py

```diff
@@ -0,0 +1,21 @@
+import sqlalchemy as sa
+from fastapi import APIRouter, Depends
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from aspen.api.authn import get_auth_user
+from aspen.api.deps import get_db
+from aspen.api.schemas.pathogens import PathogenResponse, PathogensResponse
+from aspen.database.models import Pathogen, User
+
+router = APIRouter()
+
+
+@router.get("/", response_model=PathogensResponse)
+async def list_pathogens(
+    db: AsyncSession = Depends(get_db),
+    user: User = Depends(get_auth_user),
+) -> PathogensResponse:
+    query = sa.select(Pathogen)
+    results = await db.execute(query)
+    pathogens = [PathogenResponse.from_orm(row) for row in results.scalars()]
+    return PathogensResponse.parse_obj({"pathogens": pathogens})
```

Comment:
This isn't even a nit, but I just wish we didn't need the `from_orm` and `parse_obj` boilerplate like this for so many of our responses. We have it in a bunch of places so I think it's necessary, but it feels like the serializer/schema ought to be able to handle this kind of interpretation without needing it explicitly called. Since we specify the `response_model`, it feels to me like the response model should be able to destructure whatever it gets handed based on the various attributes in the model and either send off the serialized response or raise an exception appropriately. In my perfect world, the last two lines of code would just look like
```
pathogens = [row for row in results.scalars()]
return pathogens
```

Anyway, I'm not actually requesting a change, just carping.

Reply:
no makes sense, i can try it